x-pack/filebeat/module/aws/cloudtrail/_meta/fields.yml

```diff
@@ -0,0 +1,104 @@
+- name: cloudtrail
+  type: group
```

Comment:
Could you please add `release: beta` here please? I forgot to add it for previous filesets so I just created a PR to add it: https://github.com/elastic/beats/pull/15248/files

Reply:
Added release field.  Thanks for catching this